lib/ext/adviser.rb

```diff
@@ -0,0 +1,19 @@
+require 'csv'
+
+module AdviserExtension
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def to_csv(_ = {})
+      CSV.generate do |csv|
+        csv << ['Ref. Number', 'Name', 'Firm']
+
+        find_each do |adviser|
+          csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+        end
+      end
+    end
+  end
+end
+
+ActiveRecord::Base.send(:include, AdviserExtension)
```

Comment:
If you're using this module only in one class, why include in all active record base? This means that if I call Office.to_csv they assume that the object quacks like an Adviser. I think that's was not the intended.